administrator/components/com_media/resources/scripts/components/browser/browser.vue

```diff
@@ -64,12 +67,16 @@
                 return {
                     ['media-browser-items-' + this.$store.state.gridSize]: true,
                 }
+            },
+            isModal() {
+		        return Joomla.getOptions('com_media', {}).isModal;
             }
         },
         methods: {
             /* Unselect all browser items */
             unselectAllBrowserItems(event) {
-                const eventOutside = (this.$refs.browserItems && !this.$refs.browserItems.contains(event.target)) || event.target === this.$refs.browserItems;
+                const eventOutside = (this.$refs.browserItems && !this.$refs.browserItems.contains(event.target)) &&
+                                    event.path.indexOf(this.$refs.infobar.$el) < 0 || event.target === this.$refs.browserItems;
```

Comment:
Please check the conditions here.

Reply:
i split that into two lines for better readiability

Reply:
fixed